.gitignore

```diff
@@ -25,3 +25,4 @@ __pycache__
 /*.tar
 /*.7z
 /*.tgz
+/instructions/*/
```

Comment:
This ignore line is redundant with the `.gitignore` file in `instructions`.

Reply:
True. I added the .gitignore afterwards, so that the folder appears in git.